systemd/ua-auto-attach.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Ubuntu Advantage auto attach
+After=cloud-final.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ua auto-attach
+TimeoutSec=0
```

Comment:
I think you want to add something like:

```
[Install]
WantedBy=multi-user.target
```

to get this included in the boot process.